enhancements/authentication/direct-external-oidc-provider.md

```diff
@@ -212,6 +212,59 @@ These changes have already been implemented, and the initial PR for them can be
 
 In order to use `oc` with an external OIDC provider, the tool has been [extended](https://github.com/openshift/oc/pull/1640) with the necessary functionality, including command-line arguments that enable the required configuration. In particular, [`oauth2cli`](https://github.com/int128/oauth2cli) has been vendored into the `oc` codebase. One important consideration here is that depending on the OIDC provider, further functionality might be required, in which case `oc` will have to be extended to support that too.
 
+#### authorization.openshift.io/RestrictSubjectBindings admission plugin considerations
+
+OpenShift's default admission plugin `authorization.openshift.io/RestrictSubjectBindings`
+utilizes the OAuth stack, specifically the `User` and `Group` APIs, to ensure `RoleBinding`
+subjects do not violate any restrictions in the namespace they are going to be created in.
+These restrictions are specified by `RoleBindingRestriction` resources that are served
+by the openshift-apiserver.
+
+Removing the OAuth stack entirely during external OIDC setup may result in the `authorization.openshift.io/RestrictSubjectBindings` admission plugin, and thus the KAS appearing degraded.
```

Comment:
The "appearing degraded" is because of:
- The logs stating "Unhandled Error"
- Potential to receive errors stating that the `User` and `Group` APIs are not found when trying to create `RoleBinding`s

I'm open to suggestions for better wording here since the "degraded" terminology is a bit more loaded than I had realized.

Reply:
I've updated this to try and be more explicit about the impacts a cluster administrator or users of a cluster might see that might signal to them that their cluster is not in a "healthy" state in https://github.com/openshift/enhancements/pull/1726/commits/f5c20de35657130a5dc397715bd35b529aaa483d - let me know if this reads better!